genesis_mainnet.yaml

```diff
@@ -72,9 +54,15 @@ rewarding:
     - io1z7mjef7w528nasnsafan0rp6yuvkvq405l6r8j
     - io1cup9k8hl8fp40vrj29ex8djc346780dk223end
     - io1scs89jur7qklzh5vfrmha3c40u8yajjx6kvzg9
+    - io10kyvvzu08pjeylymq4umknjal25ea3ptfknrpf
+    - io18mvepyxkcd5jkyplfqn27ydkpsendrey3xe2l8
+    - io1nz40npqa3yvek4zdasmqaetl2j4h6urejfkera
+    - io1m7p9yrejngxyvxhvn7p9g9uwlvd7uuamg8wcjd
+    - io1cwwm08dwv9phh3wt5vsdhu9gcypw9q2sc7pl9s
+    - io14aj46jjmtt83vts9syhrs9st80czumg40cjasl
   foundationBonus: "80000000000000000000"
   foundationBonusLastEpoch: 8760
-  initBalance: "1200000000000000000000000000"
+  initBalance: "200000000000000000000000000"
```

Comment:
can we later send more IOTX to the rewarding pool? is it tested?

Reply:
now gas is sent to the pool, but we don't have the cli to do it now

Reply:
Will add and verify